### PR TITLE
Update access-cluster.md

### PR DESCRIPTION
**Pull request by guzj11**
remove the excess space for a better format of the title information

### CODE REVIEW

File: content/zh/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/access-cluster.md:1]: Why this change?
Do you see extra space on this line?

Comment (guzj11) [reply content/zh/docs/tasks/access-application-cluster/access-cluster.md:1]: the tile format is revised by this pr, there is an extra space in the fronte of the '---' resulting a bad format.

Comment (tengqm) [reply content/zh/docs/tasks/access-application-cluster/access-cluster.md:1]: No, there is not such a space in the master branch.
